src/coreclr/vm/appdomain.cpp

```diff
@@ -666,11 +666,12 @@ void BaseDomain::InitVSD()
 }
 
 #ifdef HOST_WINDOWS
-void BaseDomain::InitThreadStaticBlockTypeMap()
+void BaseDomain::InitNonGCThreadStaticBlockTypeMap()
```

Comment:
This rename does not look right. Revert it?

Reply:
yes, seems I missed to rename some for linux too. Fixed them now.